### DISCUSSION TITLE
LearnPM crashes

### DISCUSSION THREAD

**Discussion by BoburaII** [LAPKB/Pmetrics#185] [Q&A]
Hi, 
I am trying to launch LearnPM shiny and it's crashing immediately on attempt to launch. I get the below warning:

Warning: Error in if: the condition has length > 1
  72: question_is_correct.learnr_radio
  70: <reactive:is_correct_info>
  54: is_correct_info
  53: observe
  52: <observer>
   9: <Anonymous>
   7: rmarkdown::run
   4: run_tutorial
   3: eval
   2: eval
   1: .rs.sourceWithProgress 

Any help with this? 
Thanks.

**Comment by mnneely** [comment]
Hmm, it is working for me. Have you tried reinstalling it?
